modules/user/user.service.js

```diff
@@ -0,0 +1,46 @@
+const userRepository = require("./user.repository");
+
+class UserService {
+    readAllUsers() {
+        return userRepository.readFileUsers();
+    }
+    async readOneUser(id) {
+        let users = await userRepository.readFileUsers();
+        let userIndex = await userRepository.findUserIndexById(users, id);
+        return users[userIndex];
+    }
+
+    async createUser(newUser) {
```

Comment:
logic about how to save user to file should be located inside of repositories.
keep in mind that your business logic don't need to know how to work with file. and your service code shouldn't be changed after we moved our project to mongo-db